rules-reviewed/jboss-eap/weblogic/weblogic-webservices.windup.xml

```diff
@@ -0,0 +1,114 @@
+        <tags>
+            <tag>reviewed-2015-05-21</tag>
+        </tags>
+    </metadata>
+    <rules>
+        <rule id="weblogic-webservices-01000">
+            <when>
+                <javaclass references="weblogic.wsee.jaxrpc.ServiceImpl">
+                    <location>INHERITANCE</location>
+                </javaclass>
+            </when>
+            <perform>
+                <classification title="WebLogic Web Service Implementation" effort="1" severity="mandatory"/>
+                <hint title="WebLogic Service Implementation Class" severity="mandatory">
+                    <message>
+                        This stub will need to be regenerated using Apache CXF.
+                    </message>
+                    <link href="https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.4/html/Development_Guide/sect-JAX-WS_Web_service_Clients.html" description="JAX-WS Web Service Clients"/>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="weblogic-webservices-02000">
+            <when>
+                <javaclass references="weblogic.wsee.async.AsyncPreCallContext"/>
+            </when>
+            <perform>
+                <classification title="Asynchronous Web Service Client" effort="8" severity="mandatory">
+                    <link href="http://docs.oracle.com/cd/E15051_01/wls/docs103/webserv_adv_rpc/asynch.html" description="WebLogic Asynchronous Execution Documentation"/>
+                    <link href="https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.4/html/Development_Guide/Develop_a_JAX-WS_Client_Application.html" description="Develop a JAX-WS Client Application"/>
+                </classification>
+                <hint title="WebLogic Asynchronous Call" effort="0" severity="mandatory">
+                    <message>
+                        The WebLogic specific Asynchronous Web Service Client API will need to be replaced with calls to the JAX-WS asynchronous API.
+
+                        More information is available in the [JBoss Enterprise Application Platform documentation](https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.4/html/Development_Guide/Develop_a_JAX-WS_Client_Application.html).
+                    </message>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="weblogic-webservices-03000">
+            <when>
+                <javaclass references="weblogic.wsee.connection.transport.http.HttpTransportInfo.setUsername({*})">
+                    <location>METHOD_CALL</location>
+                </javaclass>
+            </when>
+            <perform>
+                <hint effort="3" severity="mandatory">
+                    <message><![CDATA[
+                        Replace proprietary web-service authentication with JAX-WS standard calls. To attach authentication with JBoss, simply use the following code:
```

Comment:
Same, "JBoss" is not a product name. this should probably be rephrased:

"that provides access to the equivalent services from JBoss Enterprise Application Platform"